networkdb/cluster.go

```diff
@@ -479,26 +479,34 @@ func (nDB *NetworkDB) bulkSyncTables() {
 }
 
 func (nDB *NetworkDB) bulkSync(nodes []string, all bool) ([]string, error) {
+	syncNodes := nodes
+	retried := false
+retry:
 	if !all {
 		// If not all, then just pick one.
-		nodes = nDB.mRandomNodes(1, nodes)
+		syncNodes = nDB.mRandomNodes(1, nodes)
```

Comment:
It seems like we could achieve the same logic by just picking 2 nodes
`nodes = nDB.mRandomNodes(2, nodes)`
then exit from the sync loop as soon as one bulk is succesful
This will avoid the use of goto